<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1536828451091' stop='1536828452126'>
    <name>Login Functionality</name>
    <title>Login Functionality</title>
    <test-cases>
        <test-case start='1536828451091' status='failed' stop='1536828452125'>
            <name>User should be able to login</name>
            <title>User should be able to login</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Failed: element not visible
                  (Session info: chrome=68.0.3440.106)
                  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 10.0.17134 x86_64)</message>
                <stack-trace>ElementNotVisibleError: element not visible
                  (Session info: chrome=68.0.3440.106)
                  (Driver info: chromedriver=2.41.578737 (49da6702b16031c40d63e5618de03a32ff6c197e),platform=Windows NT 10.0.17134 x86_64)
                    at Object.checkLegacyResponse (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\error.js:546:15)
                    at parseHttpResponse (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:509:13)
                    at doSend.then.response (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\http.js:441:30)
                    at process._tickCallback (internal/process/next_tick.js:68:7)
                From: Task: WebElement.sendKeys()
                    at thenableWebDriverProxy.schedule (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:807:17)
                    at WebElement.schedule_ (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:2010:25)
                    at WebElement.sendKeys (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver.js:2174:19)
                    at actionFn (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\built\element.js:89:44)
                    at Array.map (&lt;anonymous>)
                    at actionResults.getWebElements.then (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\built\element.js:461:65)
                    at ManagedPromise.invokeCallback_ (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at asyncRun (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27)Error
                    at ElementArrayFinder.applyAction_ (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.(anonymous function).args [as sendKeys] (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
                    at ElementFinder.(anonymous function).args [as sendKeys] (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
                    at WebTextInput.sendKeys (F:\MyFolder(p.dembitskyi)\TestRepo\protractor\controls\web.text.input.js:11:47)
                    at process._tickCallback (internal/process/next_tick.js:68:7)
                From: Task: Run it("User should be able to login") in control flow
                    at UserContext.&lt;anonymous> (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:94:19)
                    at attempt (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:909:35)
                    at Spec.execute (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:526:10)
                    at UserContext.fn (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:5340:37)
                    at attempt (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4297:26)
                    at QueueRunner.run (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4217:20)
                    at QueueRunner.execute (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4199:10)
                From asynchronous test: 
                Error
                    at Suite.&lt;anonymous> (F:\MyFolder(p.dembitskyi)\TestRepo\protractor\nitro_cloud_specs\login_spec.js:10:5)
                    at addSpecsToSuite (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1107:25)
                    at Env.describe (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1074:7)
                    at describe (C:\Users\Petro.Dembitskyi\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4399:18)
                    at Object.&lt;anonymous> (F:\MyFolder(p.dembitskyi)\TestRepo\protractor\nitro_cloud_specs\login_spec.js:3:1)
                    at Module._compile (internal/modules/cjs/loader.js:689:30)
                    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
                    at Module.load (internal/modules/cjs/loader.js:599:32)
                    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>